third_party/docfx/templates/devsite/common.js

```diff
@@ -128,6 +129,23 @@ function getSummaryIntroSentence(friendlyApiName, type, name) {
     return summaryIntro;
 }
 
+/**
+ * Validates package version string.
+ *
+ * Example valid version strings:
+ * v1.0.0
+ * v11.11.11
+ * V1.0.0
+ * 1.0.0
+ * v1.0
+ * v1
+ *
+ * @param {string} version The package version to check.
+ */
+function isValidPackageVersion(version) {

```

Comment:
Is this error checking necessary? There's going to be a lot of formats we will need to consider, `v1.0.0.alpha` could be one, some versions might come as `v1.0.20.alpha-beta`; Could we simply process the versions as they come?

Reply:
Ah, because of this check, we're missing version information on languages like Python and Node:

```
"_packageVersion": "2.1.1",
```

this is a valid version specifier, but we're not including them on these languages.

Reply:
That's a good point; if there isn't a good regex to capture potential package versions, we can trust the one given in the ```docs.metadata``` file.